src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -194,13 +194,14 @@ const LoggedInHome: FC = () => {
 		setIsProcessVisitBladeOpen(true);
 	};
 
-	const mapVisitToRoom = (visit: Visit) => ({
-		image: visit.spaceImage,
-		logo: visit.spaceLogo,
-		color: visit.spaceColor,
-		name: visit.spaceName,
-		info: visit.spaceInfo,
+	const mapVisitToRoom = (visit: Visit): VisitorSpaceCardProps['room'] => ({
+		image: visit.spaceImage || null,
+		logo: visit.spaceLogo || '[ERR] No logo found',
+		color: visit.spaceColor || null,
+		name: visit.spaceName || '[ERR] No name found',
+		info: visit.spaceInfo || '[ERR] No info found',
 		slug: visit.spaceSlug,
+		id: visit.spaceId,
 	});
```

Comment:
This sets values for the VisitorSpaceCardProps['room'] object that cannot be distinguished from actual urls.
If they are undefined, set them to undefined
and make sure the code that uses them can deal with undefined values. You can default to these string values inside the component, but not inside intermediate objects that are passed around.

This feels a bit like string 'undefined' values, which can get very messy.